src/queries/authQuery.js

```
@@ -16,13 +16,17 @@ export const registerUser = async (userData, role) => {
   const endPoint = role === "customer" ? "/api/users/register/customer" : "/api/users/register/groomer";
   const formData = new FormData();
 
-  const { profileImage, ...jsonData } = userData;
+  const { profileImage, nickname, ...jsonData } = userData;
 
   delete jsonData.email;
   delete jsonData.username;
+  delete jsonData.userName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)